### PR TITLE
Update math-functions.md

### PR DESCRIPTION
**Pull request by Youssef1313**
- [x] Add methods that are new starting from .NET Core 2.1 and .NET Standard 2.1. Example is [Acosh](https://docs.microsoft.com/en-us/dotnet/api/system.math.acosh?view=netcore-3.1).
- [x] Use Console.WriteLine in examples where possible.
- [ ] Re-word confusing sentences. Example:

    > This example uses the <xref:System.Math.Cos%2A> method of the <xref:System.Math> class to return the cosine of an angle.

   For the first read, one will understand that the example calculates the cosine. However, it calculates the "Secant" using the cosine (Using the math formula: 1 / cos(x) = sec(x))

### CODE REVIEW

**Comment by Youssef1313** [comment]
> Re-word confusing sentences. Example:
>
> >  This example uses the xref:System.Math.Cos%2A method of the xref:System.Math class to return the cosine of an angle.
>
> For the first read, one will understand that the example calculates the cosine. However, it calculates the "Secant" using the cosine (Using the math formula: 1 / cos(x) = sec(x))


Can someone suggest a better wording ? I can't get up with a good one to clear the confusion.

File: docs/visual-basic/language-reference/functions/math-functions.md [hunk 1]
```diff
@@ -9,161 +9,202 @@ helpviewer_keywords:
...
+' This example produces the following output:
+' 50.3
+' 50.3
 ```  
-  
-## Example  
- This example uses the <xref:System.Math.Atan%2A> method of the <xref:System.Math> class to calculate the value of pi.  
-  
+
+## Example
```
Comment (mairaw) [inline docs/visual-basic/language-reference/functions/math-functions.md]: Can you add some sort of suffix to distinguish these headings @Youssef1313? 
Perhaps something like

 ```suggestion
## Example - Atan
```

**Comment by Youssef1313** [comment]
Done. I'm now thinking that this information isn't really visual basic-specific.
The article could be moved to another directory and add c# samples to it.

**Comment by mairaw** [comment]
Indeed @Youssef1313 and that's basically the feedback that @KathleenDollard gave offline. That this is basically repeating the System.Math page. I just think that removing this is a bigger change. So let's merge as is.

**Review by mairaw — APPROVED** [review]

**Pull request merged**